src/shared/utils/flowSchemaNormalizer.ts

```diff
@@ -65,6 +65,13 @@ export const normalizeSchema = (
   data: PipeData,
   searchTerm: string
 ): NormalizedSchema => {
+  if (!schema) {
```

Comment:
We shouldn't ever hit this, but if for some reason the schema is undefined, the page shouldn't crash